packages/cice5/package.py

```diff
@@ -24,10 +24,15 @@ class Cice5(MakefilePackage):
     # Support -fuse-ld=lld
     # https://github.com/ACCESS-NRI/spack-packages/issues/255
     variant(
-        "linker",
-        default="ld",
-        description="choose the linker program",
-        values=("ld", "lld"),
+        "ciceldflags",
+        default="none",
+        values=(
+            "none",
+            "lld",
+            conditional("flto", when="%oneapi"),
+        ),
+        multi=True,
+        description="choose none, one or multiple LDFLAGS",
```

Comment:
What happens if someone chooses `non` *and* one of the other options, e.g. `lld`?

Reply:
```
    __ldflags = {"none": "", "lld": "-fuse-ld=lld", "flto": "-flto"}
```
`none` maps to an empty string and `lld` maps to "-fuse-ld=lld". So it will inject just "-fuse-ld=lld". `none` is a suggestion from https://spack.readthedocs.io/en/latest/packaging_guide.html#multi-valued-variants .